_config.yml

```diff
@@ -54,7 +54,10 @@ collections:
     output: true
   search-plugins:
     permalink: /:collection/:path/
-    output: true  
+    output: true
+  plugin-as-service:
```

Comment:
should it be plugin-as-a-service? (Same on line 179)

Reply:
this is navigation menu, should be fine.